app/javascript/components/Projects/ProjectsList.tsx

```diff
@@ -217,3 +226,32 @@ const AutoDeployIcon: React.FC<Project> = (props) => {
     <XCircleIcon fill="red100" />
   ) : null
 }
+
+export const CriticalityTag: React.FC<Project> = ({ criticality }) => {
+  let name
+  switch (criticality) {
+    case 3:
+      name = "3: Critical"
+      break
+    case 2:
+      name = "2: Important"
+      break
+    case 1:
+      name = "1: Supported"
+      break
+    case 0:
+      name = "0: Unsupported"
+      break
+  }
+
+  return (
+    <Tags
+      tags={[
+        {
+          href: `/?criticality=${criticality}`,
```

Comment:
Shouldn't there be a `projects` in this URL?

Reply:
Root automatically resolves to projects so essentially the same.